### ISSUE TITLE
Update global disabled style

### ISSUE THREAD

**Issue by nfrgosselin** [gitcoinco/web#10007]
**User Story:**
As a Gitcoin user
I want to see consistent button states
So that I can more intuitively use the app

**Acceptance Criteria**
Disabled button state:
- 25% opacity of default state (both icon and text)
- Hover state = no hover state
- Focus state = aria-disabled (Accessibility-approved disabled state)

Disabled dropdown state:
- No open state—disabled dropdown menu shouldn’t be able to open
- 25% opacity 
- Hover state = no hover state 
- Focus state = aria-disabled

**Product & Design Links:**
Figma: "[Ticketed: Button](https://www.figma.com/file/8QjqOzDBd4HiE37YqcocK7/GR13?node-id=110%3A5601)" page -> Button states section

**Tech Details:**
The scope of this story is just to update our global design templates. We know that there are some pages with custom styles —> updating those is out of scope for this ticket. General cleanup ticket is #10009 

**Open Questions:**

**Notes/Assumptions:**

**Comment by EmHsiehGitcoin** [comment]
**Updated Acceptance Criteria**
- Disabled state = 25% opacity of default state (both icon and text)
- Disabled hover state = no hover state
- Accessibility-approved disabled state = aria-disabled

**Comment by nfrgosselin** [comment]
@ksolo heads-up that I've updated this ticket description per @EmHsiehGitcoin's notes

**Comment by nfrgosselin** [comment]
I'm not actually sure how to verify this to accept it — @thelostone-mc is there somewhere in our system that I can see this?

cc: @EmHsiehGitcoin

**Comment by EmHsiehGitcoin** [comment]
@thelostone-mc  Maybe we can create a multi-sig test grant and it'd be blocked by polygon?

**Comment by EmHsiehGitcoin** [comment]
I was able to trigger the disabled state w/ error messages on gitcoin.co live 
Looks fine to me, but can we add an "not-allowed" cursor state to disabled CTAs/links in general

![image](https://user-images.githubusercontent.com/96255151/156184882-a6d98b20-58c1-4348-af9e-7f5878fca572.png)

**Comment by thelostone-mc** [comment]
@EmHsiehGitcoin Could we push this into a new ticket saying which buttons should have the not allowed

**Comment by EmHsiehGitcoin** [comment]
will do!
